docs/src/main/paradox/java/http/compatibility-guidelines.md

```diff
@@ -30,13 +30,47 @@ dependency MUST be the same version as `akka-actor` (so the exact version from t
 In order to resolve this dependency issue, you must depend on akka-streams explicitly, and make it the same version as
 the rest of your Akka environment, for example like this:
 
-```
-val akkaVersion = "2.5.3"
-val akkaHttpVersion = "$project.version$"
-
-libraryDependencies += "com.typesafe.akka" % "akka-http_2.12" % akkaVersion
-libraryDependencies += "com.typesafe.akka" % "akka-http_2.12" % akkaVersion
-libraryDependencies += "com.typesafe.akka" % "akka-http_2.12" % akkaHttpVersion
-```
+sbt
+:   @@@vars
+    ```
+    val akkaVersion = "2.5.[...]"
+    val akkaHttpVersion = "$project.version$"
+    
+    libraryDependencies += "com.typesafe.akka" %% "akka-http"   % akkaHttpVersion
+    
+    libraryDependencies += "com.typesafe.akka" %% "akka-actor"  % akkaVersion
+    libraryDependencies += "com.typesafe.akka" %% "akka-stream" % akkaVersion
+    ```
+    @@@
```

Comment:
This does not render properly for me... I was staring at it quite some time and can't see where I made a typo in the syntax, help?